src/components/inputs/rich-text-input/dropdown.styles.js

```diff
@@ -0,0 +1,109 @@
+import { css } from '@emotion/core';
+import styled from '@emotion/styled';
+import vars from '../../../../materials/custom-properties';
+
+const getItemStyles = props => {
+  let backgroundColor = vars.colorSurface;
+  if (props.isHovered) {
+    backgroundColor = vars.colorNeutral90;
+  }
+  if (props.isSelected) {
+    backgroundColor = vars.colorAccent95;
+  }
+
+  return css`
+    background-color: ${backgroundColor};
+  `;
+};
+
+const DropdownItem = styled.div`
+  padding: 4px 8px;
+  font-size: 1rem;
+  ${getItemStyles}
+  font-family: ${vars.fontFamilyDefault};
+`;
+
+const Button = styled.button`
+  border: 0;
+  font-family: ${vars.fontFamilyDefault};
+  border-radius: ${vars.borderRadius4};
+  cursor: pointer;
+  font-size: 0.87rem;
+  padding: 4px;
+  color: ${vars.colorSolid};
+
+  &:hover {
+    background-color: ${vars.colorNeutral90};
+  }
+
+  &:focus {
+    outline: none;
+  }
+
+  &:active,
+  &:focus {
+    background-color: ${vars.colorAccent30};
+    color: ${vars.colorSurface};
+
+    * {
+      fill: ${vars.colorSurface};
+    }
+  }
+`;
+
+export const getButtonStyles = (props = { isStyleButton: true }) => [
+  css`
+    border: 0;
+    font-family: ${vars.fontFamilyDefault};
+    border-radius: ${vars.borderRadius4};
+    cursor: pointer;
+    font-size: 1rem;
+    color: ${vars.colorSolid};
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: ${props.isStyleButton ? '3px 8px' : '4px'};
```

Comment:
Why `3px`? 🤔

Reply:
Comes from design, so that the size of the input is the same as the other